app/javascript/angular/code/services/google/_map.js.erb

```diff
@@ -220,7 +221,27 @@ export const map = (
       return customMarker;
     },
 
-    clusterMarkers: function(onClick) {      
+    drawHighlightMarker: function(position) {
+      const highlightMarker = this.drawMarker({
+        position: position,
+        icon: {
+          anchor: new google.maps.Point(30, 31),
+          size: new google.maps.Size(60, 60),
+          scaledSize: new google.maps.Size(60, 60),
+          url: eval(`locationMarkerPath`)
+        }
+      });
+
+      let myoverlay = new google.maps.OverlayView();
+      myoverlay.draw = function() {
+        this.getPanes().markerLayer.id = "markerLayer";
+      };
+      myoverlay.setMap(this.mapObj);
+
+      return highlightMarker
```

Comment:
Are you using prettier? It seems strange that this does not have `;`

Reply:
I guess CI should fail too if that needs the semicolon. Can you check what's going on please?

Reply:
Prettier doesn't catch `js.erb` files.

Reply:
Aha good point. This no worries then, https://llp.kanbanery.com/projects/8040/board/tasks/2407976 will solve all problems